webpack.config.js

```diff
@@ -4,7 +4,11 @@ var webpack = require('webpack');
 module.exports = {
   entry: './src/index.js',
   output: {
-    library: 'AFRAME',
+    library: {
+      name: 'AFRAME',
+      type: 'var',
+      export: 'default'
+    },
```

Comment:
This instructs WebPack to basically ensure that when loading the UMD bundle, the default export will be available under the name AFRAME. This matches the current behaviour of the bundle AFAICT.

Reply:
Yes but we should still set `window.AFRAME = AFRAME;` ourself for the module build used with importmap.